sopel/modules/announce.py

```diff
@@ -7,18 +7,16 @@
 """
 from __future__ import unicode_literals, absolute_import, print_function, division
 
-from sopel.module import commands, example
+from sopel.module import commands, example, require_admin
 
 
 @commands('announce')
 @example('.announce Some important message here')
+@require_admin('Sorry, I can\'t let you do that', reply=True)
```

Comment:
While you're in here, should this message be updated to say why "`[the bot] can't let you do that`"? i.e., you must be an admin

Thoughts?